src/EditorFeatures/Text/TextEditorFeatures.csproj

```diff
@@ -102,7 +102,6 @@
   <ItemGroup>
     <EmbeddedResource Include="TextEditorResources.resx">
       <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>TextEditorResources.Designer.cs</LastGenOutput>
```

Comment:
Not sure how that got removed. I didn't actually add or remove anything from the project file so I'll revert the change.